app/personalised-prevention-platform/2025/10/presenting-options/index.md

```diff
@@ -0,0 +1,206 @@
+---
+title: "Presenting options"
+description: "Presented with useful and relevant opportunities, do people feel encouraged to take up activities?"
+date: 2025-10-01
+author: Mat Johnson and Roz Strachan
+tags:
+- prototyping
+---
+
+
+Hello! We’ve undergone a mild rebrand. We’re now known as the Weight Management team, still under Personalised Prevention Services.
```

Comment:
I think worth saying you'll keep the same name for posting to design histories or something?